test/extended/networking/tuning.go

```diff
@@ -0,0 +1,204 @@
+package networking
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	t "github.com/onsi/ginkgo/extensions/table"
+	o "github.com/onsi/gomega"
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/network"
+	kapiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	frameworkpod "k8s.io/kubernetes/test/e2e/framework/pod"
+)
+
+const (
+	tuningNadName = "tuningnad"
+	baseNad       = "basenad"
+	podName       = "testpod"
+)
+
+var _ = g.Describe("[sig-network][Feature:tuning]", func() {
+	oc := exutil.NewCLI("tuning")
+	f := oc.KubeFramework()
+
+	t.DescribeTable("pod should start for each sysctl on whitelist", func(sysctl, value, path string) {
```

Comment:
if the goal is just to make sure that each sysctl is accepted, could we just create a single pod that tries to set _all_ relevant sysctls?

Reply:
done